src/main/java/org/hl7/davinci/ehrserver/interceptor/OrderInterface.java

```diff
@@ -0,0 +1,15 @@
+package org.hl7.davinci.ehrserver.interceptor;
+
+import java.util.List;
+import org.hl7.fhir.r4.model.*;
+import org.hl7.fhir.instance.model.api.IBaseResource;
+
+public interface OrderInterface {
```

Comment:
This works but I feel like this pattern of having this interface and then 4 identical implementations with only the type of the resource changed isn't the best way to do this.

Reply:
I'll write another comment where I think the change could be made